From 621e4dfb32fd8f5407d5b64f3f05a21fc7d3d3fa Mon Sep 17 00:00:00 2001
From: Alexandre Rostovtsev <tetromino@gmail.com>
Date: Wed, 19 Oct 2011 10:31:17 -0400
Subject: [PATCH] Add missing comma to gnome-proxy AC_ARG_WITH

This was causing "--without-gnome-proxy" to be treated as
"--with-gnome-proxy=check".

https://bugzilla.gnome.org/show_bug.cgi?id=662203
---
 configure.ac |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure.ac b/configure.ac
index b7833d4..38d3114 100644
--- a/configure.ac
+++ b/configure.ac
@@ -66,7 +66,7 @@ dnl *** Checks for GNOME proxy backend ***
 dnl **************************************
 AC_ARG_WITH(gnome-proxy,
     [AC_HELP_STRING([--with-gnome-proxy],
-                    [support for GNOME proxy configuration @<:@default=check@:>@])]
+                    [support for GNOME proxy configuration @<:@default=check@:>@])],
     [],
     [with_gnome_proxy=check])
 AS_IF([test "x$with_gnome_proxy" != "xno"],
-- 
1.7.7

